ozaria/site/components/interactive/PageInteractive/index.vue

```diff
@@ -30,16 +32,52 @@
       }
     },
 
-    data: () => ({
-      loading: false,
-      interactive: {},
-      interactiveSession: {},
-      interactiveType: ''
-    }),
-
     computed: {
+      ...mapGetters('interactives', [
+        'currentInteractiveDataLoading',
+        'currentInteractive',
+        'currentInteractiveSession'
+      ]),
+
+      interactiveConfig () {
+        switch (this.currentInteractive.interactiveType) {
+        case 'draggable-statement-completion':
+          return this.currentInteractive.draggableStatementCompletionData
+
+        case 'insert-code':
+          return this.currentInteractive.insertCodeData
+
+        case 'draggable-ordering':
+          return this.currentInteractive.draggableOrderingData
+
+        default:
+          noty({ text: 'Interactive type is not set for the interactive', type: 'error', timeout: '2000' })
+          throw new Error('Invalid interactive type provided for interactive')
+        }
+      },
+
+      localizedInteractiveConfig () {
+        const interactiveConfigI18n = this.interactiveConfig.i18n || {}
```

Comment:
Will this also handle the i18n for interactive root properties, i.e. for promptText (https://github.com/codecombat/codecombat/blob/master/app/schemas/models/interactives/interactive.schema.js#L23)